manifests/b/BestNotes/BestNotes/1.3.2/BestNotes.BestNotes.locale.en-US.yaml

```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: BestNotes.BestNotes
+PackageVersion: 1.3.2
+PackageLocale: en-US
+Publisher: BestNotes
+PackageName: BestNotes
+License: BestNotes EULA
+Copyright: Copyright © 2019 BestNotes, LLC All rights reserved.
+ShortDescription: BestNotes Setup
+ReleaseNotes: InitialVersion

```

Comment:
```suggestion
```
Please add the official ReleaseNotes with ReleaseNotesUrl - if you cannot find them, best to omit this entry.